src/setup/demo.md

```diff
@@ -0,0 +1,236 @@
+
+### Login
+
+You can now configure your local Substra client, *with the credentials you received from us*, to connect to the `node-1` which hosts the organisation `org-1`:
+
+```sh
+# Configure the client, with a profile, a user, a password and an url
+substra config --profile <PROFILE> -u <USER> -p "<PASSWORD>" http://substra-backend.node-1.com
+
+# Login with the profile you created (located in your home folder: ~/.substra)
+substra login --profile <PROFILE>
+
+# You can then list the network nodes that are on the Substra network
+substra list node --profile <PROFILE>
+
+# And list, for example, the registered datasets in the node you are logged in
+substra list dataset --profile <PROFILE>
+```
+
+## 2. Follow Titanic example
+
+You will find everything you need to run a first end to end example with the tabular data of the Titanic: [repository](https://github.com/SubstraFoundation/substra/blob/master/examples/titanic/README.md).
+
+Please note that each resource is referenced by its hash in Substra, a unique identifier, so if you want to try out the ready to use Titanic example, you will need to edit, at least, the python files and the `description.md` for the assets (like adding a `print()` statement and editing the `description.md` files).
+
+## 3. Work with your own assets
+
+> **Note**: This is demo instance, please do not use any sensitive asset on this platform!
+
+The special [debugging section](https://github.com/SubstraFoundation/substra/tree/master/examples/debugging) can be off help preparing your asset, please have a look!
+
+### Data preparation
+
+The first step is partitioning your data, and create a test dataset and a train dataset. Following the Titanic example, you will first need to generate data samples (in `test_data_samples` and `train_data_samples` folders) by updating the `generate_data_samples` script with the path to you dataset and running it:
+
+```sh
+pip install -r scripts/requirements.txt
+python scripts/generate_data_samples.py
+```
+
+### Add dataset & objective
+
+You will now have to prepare the `opener.py` & `metrics.py` files, including their corresponding `description.md` and `Dockerfile` (where you have to add your depedencies). All assets are based on [substratools](https://github.com/SubstraFoundation/substra-tools/tree/master/substratools) classes to ease the process.
+
+The `add_dataset_objective.py` script is in charge of registering data to the `organisation`.
+
+Before running the script, please make sure you have updated:
+
+- `profile` in `client = substra.Client(profile_name="<PROFILE>")`, using the profile you created before
+- `DATASET`: `name` & `permissions` (demo organisations are named `MyOrg1MSP` & `MyOrg2MSP`)
+- `OBJECTIVE`: `name` & `permissions` (demo organisations are named `MyOrg1MSP` & `MyOrg2MSP`)
+- any different path in regard to project file structure
```

Comment:
All this is to be verified in `add_dataset_objective.py`? If yes please add it for clarity/disambiguation

Reply:
Not sure to understand @bowni, as stated, "Before running the script, please make sure you have updated:" + elements to edit.
The section is composed of 3 paragraphs: 
- edit open and metrics (+associated files)
- edit the `add_dataset_objective.py` script
- run the script

Please let me know how I could make it clearer

Reply:
Like that:

```suggestion
Before running the `add_dataset_objective.py` script, please make sure you edit it to update the following:

- `profile` in `client = substra.Client(profile_name="<PROFILE>")`, using the profile you created before
- `DATASET`: `name` & `permissions` (demo organisations are named `MyOrg1MSP` & `MyOrg2MSP`)
- `OBJECTIVE`: `name` & `permissions` (demo organisations are named `MyOrg1MSP` & `MyOrg2MSP`)
- any different path in regard to project file structure
```